docs/source/development/commands_reference.md

```diff
@@ -359,7 +359,7 @@ The above command will take the bundled `.tar.gz` file and do the following:
 `kedro micropkg pull` works with PyPI, local and cloud storage:
 
 * PyPI: `kedro micropkg pull <my-pipeline>` with `<my-pipeline>` being a package on PyPI
-* Local storage: `kedro micropkg pull <path-to-your-project-root>/dist/<my-pipeline>-0.1.tar.gz`
+* Local storage: `kedro micropkg pull <project_root>/dist/<my-pipeline>-0.1.tar.gz`
```

Comment:
```suggestion
* Local storage: `kedro micropkg pull dist/<my-pipeline>-0.1.tar.gz`
```
I assume a relative path will work here but please do give it a quick check!

Reply:
Yes, you're right, a relative path works here :).